debian/libdynamorio-clients8.0.install

```diff
@@ -0,0 +1,3 @@
+usr/lib/*/DynamoRIO-*/release/clients/*.so.*
+etc/DynamoRIO/list*
```

Comment:
OK, fair enough.  If Dr. Memory or another repository in the same Github project ends up with a system install: it could have its own name "Dr.Memory" or "DrMemory" or "drmemory" or whatnot and so I don't think we have an issue.